### ISSUE TITLE
Screen for managing players wantet

### ISSUE THREAD

**Issue by twollgam** [performous/performous#574]
### Do you want to request a *feature* or report a *bug*?

Feature

### What did you expect to see?

I want to have a new screen for managing players, where one 

- can create new players
- remove players
- change/configure players
- view some statistics

**Comment by Baklap4** [comment]
If i'm correct this can be done after the singing part already (if in score mode). One can give up a name for the score they sang.
For viewing statistics there's an open issue: #240

**Comment by twollgam** [comment]
Yes you are right with the screen after singing, but the problem is that one can not reach it from main screen.  And the workflow is reverted. I want to create players before singing and I want select players before singing. Otherwise all issues depending on player configuration will not work.

For example if a "party" game mode needs groups of players the current "workflow" is unusable.

I am not sure if the statistics in #240 are different from player statistics, but that point is like an addon and not important fot this issue.

**Comment by Baklap4** [comment]
So technically this issue should be solved with the other issue you added #572 

@performous/contributors what do you guys think of this?

I do like the idea to add playerconfiguration and assign them to mics or guitars etc. There is some overhaul to do but in the end it'll be more extendable

**Comment by twollgam** [comment]
> So technically this issue should be solved with the other issue you added #572

I would like to keep the things separated, not making issues too big.

**Comment by Baklap4** [comment]
I didn't mean this to be within #572, but as an extension upon #572 . As you said issues shouldn't be too big

**Comment by Tronic** [comment]
In a party setting, this could also work via web browser. The host enters player info, song selection etc. while guests are singing. Anyone up for developing that further?

**Comment by Baklap4** [comment]
> In a party setting, this could also work via web browser. The host enters player info, song selection etc. while guests are singing. Anyone up for developing that further?

Hmm yes this could be a solution however i still think of the webserver as a nice to have. So preferably within the UI of performous itself and later within the webinterface

**Comment by twollgam** [comment]
So, if I develop a "player configuration screen" I will take the existing player screen as "template". The data should come from and go to the Players class. There will be no player selection before singing  (this is not contained in this issue, even if I mentioned that somewhere in the comments).

**Comment by Baklap4** [comment]
Hmm as in flow i'd create a new screen based on the configured microphones or controllers where you can assign a name to the microphone/controller which uses the Player class underneath.

**Comment by Baklap4** [comment]
@twollgam how's progress coming along with #757 ? Anything you're stuck on?

**Comment by twollgam** [comment]
No, everything's fine until now. I would like to do some small things like layout etc. after release of 1.3. So 1.3 can still be a "developer" release.
